CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ All notable changes to this project will be documented in this file based on the
 
 ### Added
 
+* Added a transport class for mocking a HTTP 403 error codes, useful for testing response failures in inheriting clients
```

Comment:
Could you add a link to the PR at the end of the line?

Reply:
In terms of chronology, first came the PR into master – then the PR into the backward compatibility branch. Happy to go either way here – this is super easy

Reply:
Thus, we did not add 2 new features, just backported something. In this case I'd like to have the `[backport] ...` log message, pointing to this PR. WDYT @ruflin ?

Reply:
Both options work for me, whatever you prefer.

Reply:
Thanks for cleaning this up gents, super-fast work!